modules/agar/src/main/ts/ephox/agar/assertions/ApproxStructures.ts

```diff
@@ -49,18 +49,12 @@ export interface ElementFields {
 const elementQueue = (items: Element<any>[], container: Option<Element<any>>): ElementQueue => {
   let i = -1;
 
-  const context = () => {
-    return container.fold(() => {
-      return '\nItem[' + i + ']:' +
+  const context = () => container.fold(() => '\nItem[' + i + ']:' +
```

Comment:
Not sure what it's done here, but this is now much harder to read.

Reply:
I expanded it putting intermediate steps into variables to make it more readable.

Reply:
It was impossible to read before, imo, even with the brackets. I think it's much better now.

Reply:
Agreed, thanks @tiny-james!